### PR TITLE
opencv-python → opencv-python-headless

### PR DESCRIPTION
**Pull request by glostis-prl**
Installing the `opencv-python` wheel requires some graphical system dependencies such as `libgl1`.

The usage of OpenCV in `buzzard` is only headless, so [as recommended by the opencv-python project](https://github.com/opencv/opencv-python?tab=readme-ov-file#installation-and-usage), `opencv-python-headless` can be installed instead, and it does not require system dependencies to be installed.

### CODE REVIEW

**Review by Max19475 — APPROVED** [review]
Do we have a way to test this before merging?

**Comment by Max19475** [comment]
I do not have merge rights :disappointed: 

Maybe @ChroMaE3 ?

**Comment by ChroMaE3** [comment]
@Max19475 yes I can merge.
@Max19475 did you have an answer to your question (@glostis-prl ) ?

**Comment by glostis-prl** [comment]
@Max19475 I tried to run the tests locally (since the CI pipeline is not plugged anymore), alas to no avail…

I am however quite confident that this change is fairly harmless because `cv2` is only used in `buzzard/_a_source_raster_remap.py`, and the features of `cv2` used there (constants of the type `cv2.INTER_*` and `cv2.BORDER_*` and functions `cv2.remap` and `cv2.convertMaps`) don't seem to have anything to do with a graphical environment.

**Pull request merged**
